tests/one_to_one/tests.py

```diff
@@ -135,9 +135,11 @@ def test_unsaved_object(self):
         should raise an exception.
         """
         place = Place(name='User', address='London')
+        self.assertRaises(Restaurant.DoesNotExist, lambda: place.restaurant)
```

Comment:
Please use this style:

    with self.assertRaises(Restaurant.DoesNotExist):
        place.restaurant